Library/ViewModels/CommentsViewModel.swift

```diff
@@ -3,6 +3,8 @@ import Prelude
 import ReactiveExtensions
 import ReactiveSwift
 
+private let concurrentCommentLimit: UInt = 3
```

Comment:
Ah I left four pending comments while I was testing this (before looking at the code) and noticed the last one was not budging when I tapped it. This explains it.